machine_configuration/README.md

```diff
@@ -70,22 +70,23 @@ intel-dgpu   rendered-intel-dgpu-58fb5f4d72fe6041abb066880e112acd   True      Fa
 ```
 Ensure `intel-dgpu` MachineConfigPool is present.
 
-# Disable conflicting driver
-Run the command shown below to disable the loading of a potential conflicting driver, such as `ast` driver.
+# Disable incompatible in-tree drivers
```

Comment:
Suggest to just say
# Disable in-tree drivers

Reply:
Got it